routes/handler.go

```diff
@@ -0,0 +1,19 @@
+package routes
+
+import (
+	"github.com/gorilla/mux"
+
+	"go-rest-api/controllers"
+)
+
+const VERSION = "/api/v1"
+
+//handler requests
+func HandleRequests() *mux.Router {
+	// creates a new instance of a mux router
+	router := mux.NewRouter().StrictSlash(true)
+	router.HandleFunc(VERSION+"/items", controllers.GetItems).Methods("GET")
```

Comment:
Nice detail adding the prefix, but to avoid copying this `VERSION` constant, you could do this to get a router that already has that prefix: `router = router.PathPrefix("/api/v1/").Subrouter()`